core/controllers/suggestion_test.py

```diff
@@ -188,6 +188,52 @@ def test_create_suggestion(self):
         self.assertEqual(len(suggestions), 3)
         self.logout()
 
+    def test_edit_suggestion(self):
+
+        suggestion = self.get_json(
+            '%s?author_id=%s' % (
+                feconf.SUGGESTION_LIST_URL_PREFIX,
+                self.author_id_2))['suggestions'][1]
+
+        # Create new content for exploration after edit.
+        new_content_after_edit = state_domain.SubtitledHtml(
+            'content', 'new edit content html').to_dict()
+
+        target_type = suggestion_models.TARGET_TYPE_EXPLORATION
+        target_id = '/exp1'
+        suggestion_id = '/' + suggestion['suggestion_id']
+
+        # Change that needs to be done in submitted suggestion.
+        change = {
```

Comment:
```suggestion
        change_dict = {
```

Reply:
?? 
I don't get it

Reply:
Oh, I meant that the value assigned to this variable is a dict, so the name should indicate that. The variable should be named change_dict.

Reply:
ok, done